Userland/Libraries/LibGfx/tiff_generator.py

```diff
@@ -0,0 +1,321 @@
+import argparse
```

Comment:
missing a shebang and possibly license header

Reply:
Also want to make sure the file is committed as 0755 or w/e the other executable scripts are in the repo are, can't tell just looking at the github diff.